mantidimaging/gui/windows/live_viewer/presenter.py

```diff
@@ -45,6 +45,7 @@ def update_image(self, image_path: str):
             with tifffile.TiffFile(image_path) as tif:
                 image_data = tif.asarray()
                 self.view.show_image(image_data)
+                logger.debug("Showing image in presenter: %s", image_path)
```

Comment:
Small thing: I'd put the log message before the `with`, so that we see the message even if any of the open, load, show steps fail.